info.febvre.Komikku.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "info.febvre.Komikku",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "komikku",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gtk.vfs.*"
```

Comment:
I don't see an option to open local files, so I don't think this and `--filesystem=home` is needed.